labellab-flask/api/main.py

```diff
@@ -36,16 +36,19 @@ def register_additional_extensions(app):
     migrate.init_app(app, db)
     jwt.init_app(app)
 
+
 def register_blueprint(app):
     """Register Flask blueprints."""
     app.register_blueprint(users.usersprint, url_prefix="/api/v1")
+    app.register_blueprint(models.modelsprint, url_prefix="/api/v1")
     return None
 
+
 def register_shellcontext(app):
     """Register shell context objects."""
 
     def shell_context():
         """Shell context objects."""
-        return {"db": db, "User": models.User}
+        return {"db": db, "User": models.User, "Model": models.model}
```

Comment:
Hey, i dont think that the `models` Model requires a shell context so you may remove this :)